src/cluster.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import * as MapboxGL from 'mapbox-gl';
+import { Props as MarkerProps } from './marker';
+import supercluster from 'supercluster';
+import * as GeoJSON from 'geojson';
+
+export interface Props {
+  ClusterMarkerFactory(coordinates: number[], pointCount: number): JSX.Element;
+  clusterThreshold?: number;
+  radius?: number;
+  maxZoom?: number;
+  minZoom?: number;
+  extent?: number;
+  nodeSize?: number;
+  log?: boolean;
+  children?: Array<React.Component<MarkerProps, void>>;
```

Comment:
Consistency, I have to agree here. It's `Array<>` here and sometimes `x[]` below